frontend/src/stimulus/controllers/dynamic/meetings/export/form.controller.ts

```diff
@@ -73,4 +76,23 @@ export default class FormController extends Controller<HTMLFormElement> {
     });
     return query.toString();
   }
+
+  templatesChanged(event:Event) {
+    const target = event.target as HTMLSelectElement;
+    const data = target.options[target.selectedIndex].dataset;
+    const template = target.options[target.selectedIndex].value;
+
+    const formControl = target.closest('.FormControl') as HTMLElement;
+    const captionElement = formControl.querySelector('.FormControl-caption') as HTMLElement;
+    if (captionElement) {
+      captionElement.innerText = (data.caption || '');
+    }
```

Comment:
The currently implemented overwrite of the caption seems a misuse of captions. In my understanding, they help the user to better understand what to select between. But the way it is currently used, it is rather an explanation of the selected value. And the user only sees that after making the selection.

Maybe using radio buttons would be a better choice here:

<img width="534" height="148" alt="Image" src="https://github.com/user-attachments/assets/3439014c-391e-4573-9dd6-dafc7ac5f4a8" />

Reply:
I just adapted from the Generate PDF dialog, where it was modeled that way. But indeed, for the meetings there won't probably be more export "templates" in the future. I am using radio boxes now.